build.gradle

```diff
@@ -184,7 +184,7 @@ def versions = [
         pact_version     : '4.1.11',
         springCloud      : '3.0.3',
         junit_jupiter    : '5.8.0-M1',
-        probateCommons   : '1.3.4'
+        probateCommons   : 'DTSPB-1493-v1.1'
 ]
```

Comment:
I don't know how this is usually done - I would have thought the last version number would be increased?